Scala/Scala.sublime-syntax

```diff
@@ -969,29 +1244,50 @@ contexts:
     - match: '(?=[^ \t])'
       pop: true
 
-  for-comprehension:
-    - match: '\b(for)\s*(\{)'
-      captures:
-        1: keyword.control.flow.scala
-        2: punctuation.section.block.begin.scala
+  if-statement:
+    - match: '\bif\b'
+      scope: keyword.control.flow.scala
       push:
-        - match: '\}'
-          scope: punctuation.section.block.end.scala
+        - match: '(?=\n)'
```

Comment:
Same as...

```suggestion
        - match: '$'
```

Reply:
That's a consuming match though, as opposed to a lookahead.

Reply:
Both are non-consuming.